doc/release_notes/release_notes_1.3.rst

```diff
@@ -0,0 +1,229 @@
+.. _release_notes_1.3:
+
+ACRN v1.3 (Sep 2019)
+####################
+
+We are pleased to announce the release of ACRN version 1.3.
+
+ACRN is a flexible, lightweight reference hypervisor, built with
+real-time and safety-criticality in mind, optimized to streamline embedded
+development through an open source platform. Check out the :ref:`introduction` for more information.
+All project ACRN source code is maintained in the https://github.com/projectacrn/acrn-hypervisor
+repository and includes folders for the ACRN hypervisor, the ACRN device
+model, tools, and documentation. You can either download this source code as
+a zip or tar.gz file (see the `ACRN v1.3 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v1.3>`_)
+or use Git clone and checkout commands::
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v1.3
+
+The project's online technical documentation is also tagged to correspond
+with a specific release: generated v1.3 documents can be found at https://projectacrn.github.io/1.3/.
+Documentation for the latest (master) branch is found at https://projectacrn.github.io/latest/.
+ACRN v1.3 requires Clear Linux* OS version 31080. Please follow the
+instructions in the :ref:`getting-started-apl-nuc`.
+
+Version 1.3 major features
+**************************
+
+What's New in v1.3
+==================
+* OVMF supports Graphics Output Protocol (GOP), allowing Windows logo at guest
+  VM boot time.
+* Platform-level coordinated graceful shutdown (S5) for User VMs.
+* Virtual UART (vUART) for inter-VM communication.
+* Introduced :ref:`acrn-config <acrn_configuration_tool>` tool to configure VM
+  and hypervisor from a XML configuration file at build time.
+* Ethernet mediator now supports prioritization per VM.
+* Features for real-time determinism, e.g. Cache Allocation Technology (CAT, only supported on Apollo Lake).
+
+Document updates
+================
+We have many `reference documents available <https://projectacrn.github.io>`_, including:
+
+* Getting Start Guide for Industry scenario
```

Comment:
```suggestion
* Getting Started Guide for Industry scenario
```

Could we not already add the link, i.e. **:ref:`<rt_industry_setup>`**

Reply:
Done!